client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -307,7 +301,7 @@ const streamApis = {
 		dateProperty: 'date',
 	},
 	tag_popular: {
-		path: ( { streamKey } ) => `/read/tags/${ streamKeySuffix( streamKey ) }/cards`,
+		path: ( { streamKey } ) => `/read/streams/discover?tags=${ streamKeySuffix( streamKey ) }`,
```

Comment:
how can I test this change?

Reply:
I believe this is triggered when you click on any of the tags to the right of "Latest"

![CleanShot 2023-10-24 at 13 50 21@2x](https://github.com/Automattic/wp-calypso/assets/917632/a61133a2-aa27-4620-b33a-ff0441ccef31)

Reply:
Done, it was by going to a specific tag stream eg: https://wordpress.com/tag/lego